client/my-sites/activity/activity-log/rewind-alerts.jsx

```diff
@@ -18,27 +18,42 @@ import { requestSiteAlerts } from 'state/data-getters';
  */
 import './rewind-alerts.scss';
 
-function RewindAlerts( { alerts } ) {
-	const translate = useTranslate();
+let refreshFlag = false;
```

Comment:
I think `refreshFlag` can be component state instead of a local variable, then the site alerts will be fetched whenever it changes.

If we pass a function to set `refreshFlag` into `<ThreatAlert>`, we can call it whenever `checkFixStatus` returns, which will prompt this component to re-fetch all the alerts.

I'm getting very uneasy with this, however, because this kind of action->data->update flow should go through redux state.